api/files/views.py

```diff
@@ -336,13 +336,12 @@ def get_node(self):
     # overrides RetrieveAPIView
     def get_object(self):
         user = utils.get_user_auth(self.request).user
-
-        if (self.request.GET.get('create_guid', False) and
-                self.get_node().has_permission(user, 'admin') and
-                utils.has_admin_scope(self.request)):
-            self.get_file(check_permissions=True).get_guid(create=True)
-
-        return self.get_file()
+        file = self.get_file()
```

Comment:
very, very minor driveby: `file` is a `type` in python (which is why it gets highlighted funny like that). Better to use a different name (although it is used all over in the OSF so not a big deal if you aren't making other changes).